Fix flaky test

`/home/runner/work/RogersBankDownloader/RogersBankDownloader/Tests/RogersBankDownloaderTests/DownloadErrorTests.swift:11: error: DownloadErrorTests.testDownloadErrorString : XCTAssertEqual failed: ("The give parameters could not be converted to JSON: ["a": "b", "c": "d"]") is not equal to ("The give parameters could not be converted to JSON: ["c": "d", "a": "b"]") - `

--- a/Tests/RogersBankDownloaderTests/DownloadErrorTests.swift
+++ b/Tests/RogersBankDownloaderTests/DownloadErrorTests.swift
@@ -9,9 +9,9 @@ final class DownloadErrorTests: XCTestCase {
             "The server response contained invalid JSON: ABC"
         )
         XCTAssertEqual(
-            "\(DownloadError.invalidParameters(parameters: ["a": "b", "c": "d"]).localizedDescription)",
+            "\(DownloadError.invalidParameters(parameters: ["a": "b"]).localizedDescription)",
             """
-            The give parameters could not be converted to JSON: ["c": "d", "a": "b"]
+            The give parameters could not be converted to JSON: ["a": "b"]
             """
         )
          XCTAssertEqual(

